processes/third_party_vuln_process.md

```diff
@@ -0,0 +1,88 @@
+This document describes the management of vulnerabilities within the Node.js
+ecosystem. Vulnerabilities in Node.js core are out of this scope.
+
+# Definitions
+
+* package: in this document, a package is a module available widely to use 
+with Node.js. Most modules are hosted on the npmjs.org repository. 
+
+# Process
+
+Individuals who find potential vulnerabilities in a package are invited 
+to complete a vulnerability report on the dedicated HackerOne organization:
+
+[https://hackerone.com/nodesecurity](https://hackerone.com/nodesecurity)
+
+Vulnerabilities can also be reported by emailing <TODO: decide email address>
+
+When a potential vulnerability is reported, the following actions are taken:
+
+## Triage
+
+**Who:** The triage team
+
+**Delay:** 48 hours
+
+Within 48 hours, a member of the triage team provides a first answer to the
+individual who submitted the potential vulnerability. The possible responses
+can be:
+
+* Acceptance: what was reported is considered as a new vulnerability
+* Rejection: what was reported is not considered as a new vulnerability
+* Need more information: the triage team needs more information in order to
+evaluate what was reported.
+
+## Correction follow-up
+
+**Who:** A member of the triage team
+
+**Delay:** 45 days
+
+When a vulnerability is confirmed, a member of the triage is
+designated to follow up on this topic.
+
+With the help of the individual who reported the vulnerability, they contact
+the maintainers of the vulnerable package to make them aware of the
+vulnerability.
+
+With the package maintainer, they define a release date for the publication
+of the vulnerability. Ideally, this release date should not happen before
+the package has been patched.
+
+If the maintainers are unreachable, the vulnerability is to be made public
+45 days after the triage date.
+
+
+## Publication
+
+**Who:** A member of the triage team
+
+**Delay:** 45 days
+
+Within 45 days after the triage date, the vulnerability must be made public.
+
+If a patch is being actively developed by the package maintainer, an additional delay
+can be added with the approval of the triage team and the individual who
+reported the vulnerability (this is a simple vote where each member of the
+triage team and the vulnerability reporter have 1 vote each).
+
+The vulnerability is considered as published when a Pull Request adding it
+to this repository is opened.
+
+Within HackerOne, this is handled through a "public disclosure request".
+
+## Vulnerabilities found outside this process
+
+Vulnerabilities found and fixed outside this process should be added into
+the vulnerability base. This can be done by anyone through a Pull Request on 
+this repository.
+
+# The triage team
+
+The triage team is composed of 7 to 10 members of the security working group.
```

Comment:
Do we need a number here ?  May be better to just leave it up to the security wg team to decide when people volunteer.

Reply:
Some stated floor for the minimal number has value to me that at least expresses the intent to ensure that this is not a role that can be distilled down to an individual or lesser number who feel they can/should take it on alone. Not sure how binding this is as a process doc vs. a charter for the group which is where that would ideally be documented, but I don't think that exists atm.

Reply:
A minimum number would not be a concern for me.  I was more worried about when we hit X and then somebody volunteers.... Probably a good problem to have.   Maybe we say something like `composed of Y or more members` ..